src/pages/home/index.jsx

```diff
@@ -1,41 +1,140 @@
-      <Typography variant="h5">Home</Typography>
-
-      <Divider sx={{ m: 2 }} />
-      <Typography variant="h6">Favorite filters</Typography>
-
-      <Stack spacing={4} flexWrap="wrap" direction="row">
-        {savedQuestions.length === 0 && (
-          <Typography variant="body1">
-            You will find here all the filter setting you saved in the{" "}
-            <Link to="/questions">question game</Link>
+    <>
+    <Box sx={{ p: 2, alignItems: "center" }}>
+        <Typography
+          variant="h5"
+          sx={{ margin: "20px auto", textAlign: "center" }}
+        >
+          What game would you like to play?
+        </Typography>
+        <HomeCards />
+        {size > 0 && (
+          <Typography
+            variant="h5"
+            sx={{ margin: "40px auto", textAlign: "center" }}
+          >
+            Saved Filters
           </Typography>
         )}
+      <Stack spacing={4} flexWrap="wrap" direction="row">
         {savedQuestions.map((props) => (
           <Box sx={{ marginBottom: 5 }} key={props.title}>
             <QuestionCard showFilterResume editableTitle {...props} />
           </Box>
         ))}
       </Stack>
-    </Box>
+      </Box>
+      <Box
+        padding={{ xs: "20px", sm: "50px" }}
+        sx={{
+          backgroundColor: theme.palette.secondary.main,
+        }}
+      >
+        <Stack
+          spacing={4}
+          flexWrap="wrap"
+          direction={{ xs: "column", sm: "column", md: "row" }}
+          alignItems="center"
+          justifyContent="center"
+        >
+          <Typography
+            variant="h5"
+            sx={{
+              textAlign: "center",
+              lineHeight: "1.75",
+            }}
+          >
+            Want to make your contribution count? Create an account or sign in
+            on Open Food Facts!
+          </Typography>
+          <Stack direction="row">
+            <Button
+              variant="contained"
+              href="https://world.openfoodfacts.org/cgi/login.pl"
+              sx={{
+                backgroundColor: "white",
+                color: "black",
+                margin: "0 20px",
+              }}
+            >
+              Login
+            </Button>
+            <Button
+              variant="contained"
+              href="https://world.openfoodfacts.org/cgi/user.pl"
+              sx={{
+                backgroundColor: "white",
+                color: "black",
+              }}
+            >
+              Sign up
+            </Button>
+          </Stack>
+        </Stack>
+      </Box>
+      <Box textAlign="center">
+        <Button
+          onClick={handleOpen}
+          sx={{
+            margin: "20px auto",
+          }}
+        >
+          Learn why your contribution matters
+        </Button>
+      </Box>
+      <Modal
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
```

Comment:
this is a nice usage of aria, however it is better to avoid generic ids because the chance of repeating them is higher and is not very clear to what they are referring to. May be rename to something  like 'modal-contribution-title' and 'modal-contribution-information'